INSTALLATION.md

```
@@ -274,31 +302,32 @@ When creating substantial content, briefly note which memories informed the appr
-
-
 ![Claude Desktop Using Memory](screenshots/claude-desktop-with-instructions.jpg)
-*Claude automatically using memory tools with custom instructions*
+_Claude automatically using memory tools with custom instructions_
 
 **Example behavior:**
 
-*Without the prompt:*
+_Without the prompt:_
+
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)